<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #B5B8B1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="field" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>200</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid lightgray;
background: #B5B8B1;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string>SPEED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string>LEVEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreNum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="levelNum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speedNum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid lightgray;
background:  #A5A5A5;
text-align: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
